### PR TITLE
[AiLab] make autogenerated design elements visible automatically

### PR DESCRIPTION
**Pull request by Erin007**
When students import a machine learning model into App Lab, the autogenerated design elements didn't display until the user hit run. With this change, it is readily apparent that design elements were created for the user because we now show them without the student needing to the run the program. 

BEFORE

https://user-images.githubusercontent.com/12300669/109999535-537ab000-7ce0-11eb-9f88-c19c8881993a.mov

AFTER 

https://user-images.githubusercontent.com/12300669/109999374-275f2f00-7ce0-11eb-9ab7-feb469abe5b9.mov

### CODE REVIEW

File: apps/src/applab/designMode.js [hunk 1]
```diff
@@ -1043,6 +1043,13 @@ designMode.onInsertEvent = function(code) {
   Applab.scrollToEnd();
 };
 
+designMode.onInsertAICode = function(code) {
+  Applab.appendToEditor(code);
+  getStore().dispatch(actions.changeInterfaceMode(ApplabInterfaceMode.DESIGN));
```
Comment (maddiedierker) [inline apps/src/applab/designMode.js:1050]: interesting... do you know why we have to switch to design mode for this to work?

Comment (Erin007) [reply apps/src/applab/designMode.js:1050]: This is my understanding, but I'm still not 100% clear on all the intricacies of App Lab... 

Design elements are typically created:  1.) by the user in design mode or 2.) as starter elements from the levelbuilder via the `start_html` property 

In [applab.js,](https://github.com/code-dot-org/code-dot-org/blob/d8fc97b757583e073a72fb82a17072eb631da901/apps/src/applab/applab.js#L317) we only get the HTML if designModeViz element is visible. 
```
Applab.getHtml = function() {
  // This method is called on autosave. If we're about to autosave, let's update
  // levelHtml to include our current state.
  if ($('#designModeViz').is(':visible')) {
    designMode.serializeToLevelHtml();
  }
  return Applab.levelHtml;
};
```

When there is starter HTML from levelbuilder we set the HTML [when the App is initialized](https://github.com/code-dot-org/code-dot-org/blob/d8fc97b757583e073a72fb82a17072eb631da901/apps/src/applab/applab.js#L548). 

In this case, I'm adding design elements outside the context of design mode and outside the starterHtml code path, so the easiest way to get those elements to be rendered is to briefly switch into design mode, making the designModeViz element visible, thus popping into the conditional that will set the the level html and render the autogenerated elements.

Comment (maddiedierker) [reply apps/src/applab/designMode.js:1050]: ah, that makes sense. could you add a comment above this line that explains why we have to switch to design mode here?

**Review by maddiedierker — APPROVED** [review]

File: apps/src/applab/designMode.js (hunk 1 above)
Comment (Erin007) [reply apps/src/applab/designMode.js:1050]: yep - will do!

**Review by breville — APPROVED** [review]

**Pull request merged**